src/services/errors/exception.filter.ts

```diff
@@ -1,23 +1,54 @@
 import { NextFunction, Request, Response } from 'express';
 import { IExceptionFilter } from './exception.filter.interface';
-import { HTTPError } from './http-error.class';
+import { Exception } from './exception.class';
 import { inject, injectable } from 'inversify';
 import { ILogger } from '../logger/logger.interface';
 import { TYPES } from '../../constants/constants';
 import 'reflect-metadata';
 import { StatusCodes } from 'http-status-codes';
+import { JsonWebTokenError } from 'jsonwebtoken';
+import { Prisma } from '@prisma/client';
 
 @injectable()
 export class ExceptionFilter implements IExceptionFilter {
 	constructor(@inject(TYPES.ILogger) private logger: ILogger) {}
 
-	catch(err: Error | HTTPError, req: Request, res: Response, next: NextFunction): void {
-		if (err instanceof HTTPError) {
-			this.logger.error(`[${err.context}] Error ${err.statusCode}: ${err.message}`);
+	catch(err: Error | Exception | JsonWebTokenError, req: Request, res: Response, next: NextFunction): void {
```

Comment:
We agreed that you would simplify this handler as much as possible and not complicate it. In all cases, the error has a name,code, and message. Why not show a uniform error?